repos/system_upgrade/el7toel8/libraries/fetch.py

```diff
@@ -21,6 +22,34 @@ def _raise_error(local_path, details):
     raise StopActorExecutionError(summary, details={'details': details, 'hint': hint})
 
 
+def _request_data(service_path, cert, proxies, timeout=REQUEST_TIMEOUT):
+    logger = api.current_logger()
+    attempt = 0
+    while True:
+        attempt += 1
```

Comment:
```suggestion
    for attempt range(1, MAX_ATTEMPTS + 1):
```

Reply:
I prefer to keep it stupid, They way it's written it's clear there are just two possibilities how the function ends, without need to think about that.